### PR TITLE
B2BCHCKOUT-10

### PR DESCRIPTION
**Pull request by arturmagalhaesjr**
### Feature 

Created a quote button which goes to the flow /b2b-quote/create

### How to test

Workspace: https://artur2--b2bstoreqa.myvtex.com/

1) Access the URL above
2) Login with this credentials: kevin.yee@vtex.com / Vtex1234
3) Add some product to cart
4) Click on the mini cart and then click on Go to Checkout
5) You should see the "Create Quote" below from "Proceed to Checkout"
6) Click on the button and then you will be redirected to "B2B Create quote URL"

Also, you can navigate through checkout by clicking "Proceed to Checkout" to see the button on payment section as well.

Recording: https://drive.google.com/file/d/1vGGhMZhpgJ-pIdr4oxRsPd-NiY9P2Mmj/view?usp=sharing

### CODE REVIEW

File: checkout-ui-custom/checkout6-custom.js [hunk 1]
```diff
@@ -6,7 +6,8 @@
   let checkVtex = null
   const translation = {
     'pt-BR': {
-      cartPurchaseOrderLabel: 'Número do Pedito',
+      cartPurchaseOrderLabel: 'Número do Pedido',
+      createQuoteButtonLabel: 'Criar uma Cota',
```
Comment (wender) [inline checkout-ui-custom/checkout6-custom.js:10]: Criar uma Cotação

**Review by wender — CHANGES_REQUESTED** [review]

**Review by arturmagalhaesjr — COMMENTED** [review]
Done

**Review by polishq — CHANGES_REQUESTED** [review]
I think we want to add this button conditionally, perhaps via a toggle in the settings, since the account may or may not have the quotes app installed.

**Review by arturmagalhaesjr — COMMENTED** [review]
done

**Review by arturmagalhaesjr — COMMENTED** [review]
Done

**Review by polishq — DISMISSED** [review]

**Review by wender — DISMISSED** [review]

**Pull request merged**